package.json

```diff
@@ -81,7 +81,7 @@
         },
         {
           "command": "gather.smartSelect",
-          "when": "notebookType == jupyter-notebook && notebookCellType == code",
+          "when": "notebookType == jupyter-notebookxxx && notebookCellType == code",
```

Comment:
Is this xxx to disable smart select? Intentional?

Reply:
yes :/
there's work to be done in the jupyter extension (actions for multiselected cells) for it to be meaningful